python/sdk/docs/README.md

```diff
@@ -9,7 +9,7 @@ pip install -r requirements_docs.txt
 
 Install `pandoc`
 ```bash
-conda install -c conda-forge pandoc=1.19.2
```

Comment:
`-c conda-forge` doesn't need to be specified explicitly anymore as it's the default channel.

Reply:
Technically we don't need to change this right? Since this is going to be run by the users themselves and we don't really know or care about which repository they're going to use 🤔